src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,30 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+    ArrayList<DriverCommand> commands = new ArrayList<>();
```

Comment:
Should be private!
List type would be sufficient and would not limit the implementation.